data/announcements/scheduled.yaml

```diff
@@ -203,7 +203,7 @@ announcements:
     color: #fffff;
   title: |
     <img src ="/images/announcements/kccnc-na-2023-white.svg" style="float: right; height: 80px;" />
-    <a href= "https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/">KubeCon + CloudNativeCon NA 2023</a> <em> Chicago, Illinois + Virtual</em>.
+    <a href= "https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/">KubeCon + CloudNativeCon NA 2023</a> <em> Chicago, Illinois + Virtual</em>
```

Comment:
Why this change?

Reply:
That's uncalled for, I'll revert this

Reply:
Please do update this PR; ideally, also rebase it against _main_.